index.js

```diff
@@ -68,7 +69,44 @@ function toggleColor(element) {
   }
 }
 
+const h1Header = document.querySelector("#header")
+// h1Header.addEventListener("click", toggleColor)
+// won't work because toggleColor expects an element but event listener will pass an event to the callback
+h1Header.addEventListener("click", function () {
+	toggleColor(h1Header)
+})
+
 /***** Deliverable 2 *****/
 
+const likeBtn = document.querySelector(".like-button")
+
+likeBtn.addEventListener("click", function() {
+	const likesCnt = document.querySelector(".likes")
+	player.likes++
+	likesCnt.textContent = player.likes + " Likes"
+})
 
 /***** Deliverable 3 *****/
+
+const newGoalForm = document.querySelector("#new-goal-form")
+
+newGoalForm.addEventListener("submit", handleNewGoalFormSubmit)
+
+let id = player.goals[player.goals.length -1].id + 1
+
+function handleNewGoalFormSubmit(event) {
+	event.preventDefault()
+
+	const link = event.target.link.value
+	const description = event.target.description.value
+
+	const goal = {
+		link: link,
+		description: description,
+		id: id++
+	}
```

Comment:
If you want to save on variables declarations, you could also create your object like so:
```javascript
  const newGoal = {
    id: player.goals[player.goals.length -1].id + 1,
    link: event.target.link.value,
    description: event.target.description.value
  }
```

Great work!